studies/migrations/0063_auto_20200428_1253.py

```diff
@@ -0,0 +1,90 @@
+# Generated by Django 2.0 on 2020-04-28 16:53
```

Comment:
See comment below about naming your migrations. Also, try to "squash" your migrations - if you find more changes are being made,`python manage.py migrate ${number of previous}`, delete the old migration file, and then re-run `python manage.py makemigrations --name=${whatever the name is}` and then finally run `python manage.py migrate` again